### ISSUE TITLE
JS exception when creating Kubernetes service using GUI

### ISSUE THREAD

**Issue by TimAltman** [mesosphere/dcos-kubernetes-quickstart#24]
I can't create kubernetes service (1.7.6-1.7.7) using GUI in DC/OS 1.10.0. However Iwas able to deploy kubernetes 1.7.5 before.
My steps are as follows: Services->Run a Service->Install a Package->beta kubernetes.
When I click Configure or Deploy nothing happens. In web console I see errors:
for Configure
```
tracker.js:38 Uncaught Error: TrackJS Caught: Cannot convert undefined or null to object
    at keys (<anonymous>)
    at SchemaUtil.js:279
    at Array.forEach (<anonymous>)
    at Object.schemaToMultipleDefinition (SchemaUtil.js:269)
    at t.value (SchemaForm.js:245)
    at t.value (SchemaForm.js:49)
    at h.performInitialMount (vendor.574d2ad67f110a649eab.js:sourcemap:18)
    at h.mountComponent (vendor.574d2ad67f110a649eab.js:sourcemap:18)
    at Object.mountComponent (vendor.574d2ad67f110a649eab.js:sourcemap:3)
    at p.mountChildren (vendor.574d2ad67f110a649eab.js:sourcemap:18)
```
for Deploy
```
tracker.js:38 Uncaught Error: TrackJS Caught: Cannot convert undefined or null to object
    at keys (<anonymous>)
    at SchemaUtil.js:279
    at Array.forEach (<anonymous>)
    at Object.schemaToMultipleDefinition (SchemaUtil.js:269)
    at t.value (SchemaForm.js:245)
    at t.value (SchemaForm.js:49)
    at h.performInitialMount (ReactCompositeComponent.js:351)
    at h.mountComponent (ReactCompositeComponent.js:258)
    at Object.mountComponent (ReactReconciler.js:46)
    at p.mountChildren (ReactMultiChild.js:238)
```

**Comment by pires** [comment]
This is an issue with DC/OS 1.10 that has been fixed in the next release, 1.10.1. We're sorry for any inconvenience and recommend you use the CLI for now.

**Comment by pires** [comment]
We've released an interim solution in [0.2.2-1.7.7-beta](https://docs.mesosphere.com/service-docs/beta-kubernetes/0.2.2-1.7.7-beta/).
